frontend/gwms_renew_proxies.py

```diff
@@ -213,9 +221,18 @@ def main():
     proxies.remove('COMMON')  # no proxy renewal info in the COMMON section
     for proxy_section in proxies:
         proxy_config = dict(config.items(proxy_section))
+        if 'rfc' not in proxy_config:
+            proxy_config['rfc'] = True
+        else:
+            if safe_boolcomp(proxy_config['rfc'], False):
+                proxy_config['rfc'] = False
+            else:
+                proxy_config['rfc'] = True
+        if 'path_length' not in proxy_config:
+             proxy_config['path_length'] = '20'
```

Comment:
I don't think you need this block new block because we load the defaults here:

https://github.com/glideinWMS/glideinwms/blob/3eb90b1901924bf25c75f9b486bc7aeddd2bd0cc/frontend/gwms_renew_proxies.py#L198-L200